dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnable.kt

```diff
@@ -125,9 +177,20 @@ internal class DataUploadRunnable(
         return status
     }
 
+    private fun sendUploadQualityEvent(category: UploadQualityCategories, specificType: String? = null) {
+        rumFeature?.sendEvent(
+            UploadQualityEvent(
+                track = featureName,
+                category = category,
+                specificType = specificType
+            )
+        )
+    }
+
     // endregion
 
     companion object {
         internal const val LOW_BATTERY_THRESHOLD = 10
+        private const val HTTP_SUCCESS_CODE = 202
```

Comment:
minor: 202 is `Accepted`. This is as well how it is named in `DataOkHttpUploader.HTTP_ACCEPTED`